.github/workflows/windows_test_comment_trigger.yml

```diff
@@ -7,7 +7,7 @@ on:
 jobs:
   windows_test_preparation:
     runs-on: ubuntu-20.04
-    if: ${{ github.event.issue.pull_request && github.event.comment.body == '/testwin' }}
+    if: ${{ false && github.event.issue.pull_request && github.event.comment.body == '/testwin' }}
```

Comment:
Disabling part of the jobs doesn't seem like a good solution. I would rather disable the whole workflow either manually (after this PR merge) or programmatically modifying the lines 3-5.
I also talked to @romanovvlad and we agreed that windows_test_completion should also be skipped if `windows_test_preparation` and `windows_default` jobs are skipped.

Most of the time the workflow execution looks like this - https://github.com/intel/llvm/actions/runs/4633598850. We run only `windows_test_completion`, which doesn't do anything useful as most of the comments are about reviewing the code rather than running the tests on windows.

Reply:
Okay, you can manually disable the workflow after the PR merge.
Also, regarding `windows_test_completion` workflow, I've removed the `if: always()` statement so this workflow will also gets skipped if the `windows_default` and `windows_test_preparation` workflows are skipped.

Reply:
> Also, regarding `windows_test_completion` workflow, I've removed the `if: always()` statement so this workflow will also gets skipped if the `windows_default` and `windows_test_preparation` workflows are skipped.

FYI. It was added for reporting the status of failed jobs (see https://github.com/intel/llvm/commit/a1fbd64649153161e40c00aa84cf923f14506266). I think the right way to fix this is to replace `needs: [windows_test_preparation, windows_default]` with something like `if: jobs.windows_test_preparation.result != skipped` (syntax to be checked). Otherwise `needs:` will run the status reporting job only if testing was successful.

This is not a blocking issues as we are going to disable this job anyway, but if our experiments with pre-commit will be unsuccessful, we will need to fall back to on-demand testing and we need accurate reporting.

Reply:
Sounds good to me. Thanks for clarifying.